product_docs/docs/edbcloud/beta/migration/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Migrating databases to EDB Cloud
+---
+
+EDB provides migration tools to bring data from Oracle, PostgresSQL, and EDB Postgres Advanced Server databases into EDB Cloud. These tools include Migration Portal and Migration Toolkit for Oracle migrations. 
+
+
+
+## Migrating from Oracle
+
+The [Migration Portal documentation](../../../migration_portal/latest) provides the details for executing the migration steps using Migration Portal: 
+
+ 1. [Schema extraction](../.../../migration_portal/latest/04_mp_migrating_database/01_mp_schema_extraction/)
+ 1. [Schema assessment](../../../migration_portal/latest/04_mp_migrating_database/02_mp_schema_assessment/)
+ 1. [Schema migration](../../../migration_portal/latest/04_mp_migrating_database/03_mp_schema_migration/)
+ 1. [Data migration](../../../migration_portal/latest/04_mp_migrating_database/04_mp_data_migration/)
+   
+The Migration Portal documentation describes how to use Marketing Toolkit for the data migration step. This is a good option for smaller databases. 
+
+## Migrating from Postgres
+For a simple way to import an existing PostgresSQL or EDB Postgres Advanced Server database into EDB Cloud, see [Importing an existing Postgres database](cold_migration).
```

Comment:
Perhaps we should also indicate that Migration Toolkit can be used to migrate Postgres databases to EDB Cloud here.  Would something like the following work?

> There are several options available for migrating EDB Postgres Advanced Server and PostgreSQL databases to EDB Cloud.  One option is to use the EDB Migration Toolkit.  Another simple option for many use cases is to import an existing PostgreSQL or EDB Postgres Advanced Server ...

Reply:
Done